dnf-behave-tests/dnf/security-advisory.feature

```diff
@@ -64,6 +116,15 @@ Scenario: upgrade-minimal with pkgs specified sec-severity
         | Action        | Package                   |
         | upgrade       | advisory_B-0:1.0-2.x86_64 |
 
+Scenario: check-upgrade --minimal with pkgs specified sec-severity
+   When I execute dnf4 with args "upgrade-minimal advisory_B --sec-severity Moderate"
```

Comment:
I believe the `I execute dnf4` step shouldn't be here, it is not being run, it is a wrong command and these tests are only for dnf5.

Reply:
fixed